crates/red_knot/README.md

```diff
@@ -0,0 +1,27 @@
+# Red Knot
+
+Red Knot is an extremely fast type checker.
+Currently, it is a work-in-progress and not ready for user testing.
+
+Red Knot is designed to prioritize good type inference, even in unannotated code,
+and aims to avoid false positives.
+It will *not* be a drop-in replacement for either Mypy or Pyright.
+
+
+## Contributing
+
+The crate structure is as follow:
+
+* `red_knot`: Command line interface
+* `red_knot_project`: Project discovering
+* `red_knot_python_semantic`: Core type checking
+* `red_knot_server`: Language server implementation
+* `red_knot_test`: Type inference test framework
+* `red_knot_vendored`: Public Red-Knot-specific Python APIs
```

Comment:
I worry that this might get easily out of date. We're still at the stage where we're often doing quite sweeping reorganisation of code. Also, you're missing `ruff_db` from the list :-)

Reply:
I'd prefer to remove it as well. Most of it is either obvious from the name or documented in the project READMEs.